flex/codegen/src/hqps/hqps_scan_builder.h

```diff
@@ -79,7 +83,7 @@ class ScanOpBuilder {
   // get required oid from query params
   ScanOpBuilder& queryParams(const algebra::QueryParams& query_params) {
     if (!query_params.has_predicate()) {
-      throw std::runtime_error(std::string("expect expr in params"));
+      LOG(WARNING) << "No expr in params";
```

Comment:
The warning log is not necessary as it is common to have no parameter

Reply:
Use `VLOG(10)` instead.